scripts/testing/scan_for.sh

```diff
@@ -15,8 +15,16 @@ outfile=timed_scan_$(date  +"%d.%m.%y-%H.%M").csv.gz
 
 echo "
 $(tput setaf 2)
-        Scanning for $duration, output file is $outfile .
+        Scanning for $duration, expect a $outfile.png in $(pwd) afterwards.
 $(tput sgr0)"
 
 $scriptpath/start_scanning.sh $outfile
-nohup bash -c "sleep $duration; $scriptpath/stop_and_finalize_scanning.sh $outfile" 2> /dev/null &
```

Comment:
There was actually another bug hidden, because
1. `2>` just redirects stderr, not stdout
2. the redirection happens for the finalizing script, not for nohup itself